src/EnergyPlus/IceThermalStorage.cc

```diff
@@ -2360,6 +2357,33 @@ namespace IceThermalStorage {
         return CalcDetIceStorLMTDstar;
     }
 
+    Real64 CalcQstar(int const CurveIndex,      // curve index
```

Comment:
Nice to move this to a separate function.

Reply:
@mjwitte Thanks!  Makes unit test creation much cleaner as well.